_pages/qa/bookit.md

```diff
@@ -1,21 +1,27 @@
 ---
 layout: "q-and-a"
 permalink: "/bookit/"
-title: Using BookIT!
+title: How do I book a desk at a GSA building?
 ---
 # {{ page.title }}
 
-## Configure BookIT! to save yourself time in the future
+We use [BookIT!](http://bookit.gsa.gov/) to book a desk or meeting room in GSA buildings.
+
+## How to Use BookIt
 
-[BookIT!](https://bookit.gsa.gov) allows you to reserve a meeting room in any GSA building ahead of time.
 
-If you are having login issues, try bookmarking [this address](https://bookit.gsa.gov/mobile/auth/spnego/spnegoLogin.jsp). It should immediately ask for your ENT password.
 
 ## Set your default region and office building
-* This will make it so that BookIT! remembers your basic preferences, and you won't have to scroll through drop-down menus every time you need to reserve a meeting room.
 
-## Add delegates
-* Add people from your team and a couple other people from ops as your delegates in BookIT!
+* BookIT! can remember your office location, and you won't have to scroll through drop-down menus every time you need to reserve a meeting room.
+
+## Troubleshooting BookIT!
+
+If you are having login issues, try bookmarking [this address](https://bookit.gsa.gov/mobile/auth/spnego/spnegoLogin.jsp). It should immediately ask for your ENT password.
```

Comment:
I know this was in the original, but ["click here"-style links aren't accessible](https://pages.18f.gov/accessibility/links/). Perhaps change it to `[the BookIT! login page](https://...)`?